README.md

```diff
@@ -577,6 +579,70 @@ $data = [
 
 > [1, 2, 3, 3, 4, 5]
 
+### IfElse
+
+Delegates to one expression or another depending on whether the specified condition strictly evaluates to true.
+
+If the condition does not return a boolean an InvalidReturnException will be thrown.
```

Comment:
* Since this exception was renamed this documentation appears incorrect.
* Prefer *is thrown* to *will be thrown*  for brevity.
* Classes should be demarcated with back-ticks (`` ` ``) to indicate they are classes.

Consider simply copying the following example.

```
If the condition does not return boolean, `InvalidConditionException` is thrown.
```